zebra/rt_netlink.c

```diff
@@ -4309,13 +4309,16 @@ static int netlink_ipneigh_change(struct nlmsghdr *h, int len, ns_id_t ns_id)
 						    zif->link_ifindex);
 		if (!link_if)
 			return 0;
-	} else if (IS_ZEBRA_IF_BRIDGE(ifp))
+	} else if (IS_ZEBRA_IF_BRIDGE(ifp)) {
 		link_if = ifp;
-	else {
+	} else {
 		link_if = NULL;
+		char ip_buf[ETHER_ADDR_STRLEN];
```

Comment:
Please move this above `link_if` and a newline after variables (same is suggested by frrbot/verify check).